### DISCUSSION TITLE
Best Training Augmentation for Text Objects ?

### DISCUSSION THREAD

**Discussion by xxniiinxx** [ultralytics/yolov5#12738] [General]
Hello Everybody

I have a Custom Model with 14 classes, each class is a text object, basically they are small double digit numbers. My model is trained on 150 images, however the overall precision is still very low. As I'm adding more images I can see some improvement but I believe the improvement rate is very low, due to training and not proper Augmentation setup for my usecase detecting small text objects. I would appreciate some Augmentation advice for my usecase.

**Comment by jmarintur** [comment]
Hi @frankomondo, I used in the past [albumentations](https://github.com/albumentations-team/albumentations), it's a great way to start with. Regarding your dataset, I don't know if you have the ability of creating more data using different fonts perhaps. You could also try eroding/dilating the digits. Good luck, it sounds like an interesting use case.
